2016-07/july-08.md

```diff
@@ -0,0 +1,63 @@
+* [Jesus Rodriguez](https://github.com/foxandxss)
+* [Spencer Stark](https://github.com/spencerstark) 
+
+#### Attendees Notes
+
+* Jesus and Spencer joined us today for the Google hangout. Jesus is working directly with Juho for documentation on webpack/webpack.io and his involvement was pertinent. Spencer is helping Sean take documentation and meeting notes.
+
+### Agenda
+
+#### Medium and Publications
+
+* [The initial post about webpack on medium](https://medium.com/webpack/webpack-its-getting-real-92c60fca1db1#.8nv2qek91) was really well received. Juho said that week’s meetings notes should accompany a publication. Everyone was in agreement. We will add it to the checklist of things we'll make sure is included when we post our weekly publication. Sean suggested to continue to publications on Saturday Morning (CST) and everyone was happy with this. 
+
+* The next article will be on documentation, which is in review now. It will contain a call to action for involvement in the webpack/webpack.io documentation process. We will also want to thank [people for offering their time maintaining Plugins and Loaders](https://github.com/webpack/webpack/issues/2734) which has already had some great feedback.
+
+#### Documentation
+
+* Antwar open source library for documentation. Tobias mentioned previously that we will have a separate page in documentation that will support dynamic content (JavaScript). We will have to wait and see what ideas he is cooking up for documentation tools/utils. 
+
+#### Slack
+
+* Sean mentioned starting a webpack slack org. We tossed the idea around about tying out slack and decided we will pilot the slack org that Sean has. If it is successful, we could potentially open this up to people to generate invites to (like slackr) and join.
+
+#### Core Loaders and Plugins
+
+* There was awesome response from people. Sean suggested we have two to three people per repository. We will have to sort through the list of pledges and assign to the best of our ability those who are interested (after being vetted).
+
+* We discussed privileges and pull requests in regards to these loaders and we thought best to wait until teams are solidified before creating a standard. 
+	
+* The first work that we will call on maintainers will be to mimic issue templates / types from webpack/webpack (where relevant) so that issues are high quality across the board. We have seen a huge influx in HIGH QUALITY github issues, and they are triaged and fixed easier because of this. 
+	
+  * This will be some of the first “Pilot Pull Requests” for these teams once created.
+	
+  * They will also need to review old pull requests in repositories, start to implement support for webpack 2.0 breaking changes.
+	
+  * Continue to monitor and support once established. 
+
+#### Sponsorship
+* Still looking at options
+	
+* Needing to find a list of organizations that utilize webpack as a core tool (IE. Trivago, Webflow, etc.)
```

Comment:
Instagram's been using Webpack for a long time, Pete Hunt's howto guide (https://github.com/petehunt/webpack-howto) was written back when Instagram was just getting started with Webpack :)